src/common/components/repositories-list/index.js

```diff
@@ -100,15 +99,11 @@ class RepositoriesList extends Component {
   }
 
   render() {
-    const isLoading = this.props.snaps.isFetching;
+    const { snaps } = this.props.snaps;
+    const hasSnaps = (snaps && Object.keys(snaps).length > 0);
 
     return (
       <div className={styles.repositoriesList}>
-        { isLoading &&
-          <div className={ spinnerStyles }>
```

Comment:
I guess this throws on linter, as `spinnerStyles` are still imported but not used.